source/_integrations/honeywell.markdown

```diff
@@ -28,7 +28,19 @@ If your system is compatible with this integration, then you will be able access
 
 ## Supported hardware
 
-Any US-based Honeywell thermostat the uses [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/)
+Home Assistant is integrated with the following devices through [https://mytotalconnectcomfort.com/portal/](https://mytotalconnectcomfort.com/portal/):
+
+- Thermostats
+  - Every thermostat is exposed as a `climate` entity
+  - Example devices: TH6320R1004
+- Sensors
+  - A Temperature `sensor` entity.
+  - A Humidity `sensor` entity.
+  - Example devices: C7089R1013
+
+Others devices like Security systems are not currently supported by this integration.
+
+## Configuration
```

Comment:
The `##configuraiton` header comes with the linked part (just below).  Putting in this one results in two CONFIGURATION headers :( 
The TOC should update when it's built server side?

Reply:
Ahh good to know on the Config header being included. For the TOC, I'm not actually sure if it will be smart enough to include the Config section. Could try running the site locally to see how it renders.

Reply:
It doesn't render locally, running local how I caught the double header.
Maybe the "master" reviewer knows how to do TOC with the linked header.

Also, Frenck commented earlier saying "this isn't code" on the `honeywell` tags.  I'll remove marks you put in because of that.